compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/tree/TreeTraversalAPITest.java

```diff
@@ -66,6 +66,22 @@ public void testFindTokenFromPosition() {
         Assert.assertEquals(actualToken.toString(), expectedLexeme);
     }
 
+    @Test(enabled = false)
```

Comment:
We can enable these test cases by calculating the positions on the fly. Think of a way to get the positions during the runtime by using `String.indexOf()` methods

Reply:
Added the change